tools/docker-go

```diff
@@ -18,6 +18,9 @@ Required:
     --sdk-image                 Name of the SDK image to use
     --go-mod-cache              The Go module cache path to mount into the container
     --command                   The command to run in the SDK container
+
+Optional:
+	--additional-docker-args    Additional free-form arguments to provide to "docker run" (i.e. mounts, env-vars, etc.)
```

Comment:
Indentation is a little off here. Minor thing if you need to do another update.

Reply:
Hmmm weird, nvim inserted a tab here instead of 4 spaces. But yes, good catch! Will update with other feedback